backend/inspirehep/search/aggregations.py

```diff
@@ -252,6 +252,37 @@ def hep_rpp(order, title="Exclude RPP", agg_type="checkbox"):
     }
 
 
+def hep_self_author_claimed_papers_aggregation(
+    order, title="Claims", agg_type="checkbox",
+):
+    return {
+        "self_curated_relation": {
+            "filters": {
+                "filters": {
+                    "Claimed papers": {
+                        "nested": {
+                            "path": "authors",
+                            "query": {"term": {"authors.curated_relation": True}},
+                        }
+                    },
+                    "Unclaimed papers": {
+                        "nested": {
+                            "path": "authors",
+                            "query": {"term": {"authors.curated_relation": False}},
+                        }
+                    },
+                }
+            },
+            "meta": {
+                "title": title,
+                "order": order,
+                "type": agg_type,
+                "is_filter_aggregation": True,
```

Comment:
After adding this only one checkbox shows (for Claimed papers) and without it there are two checkboxes but when you click on one the other is checked too.

Reply:
what's the request url when you click one of the two?

Reply:
this looks very similar to the seminars thing, so I don't see why it doesn't work here.

Reply:
Without `"is_filter_aggregation": True` the request doesn't contain this filter. With it it seems to be correct (but only for Claimed papers)